cmd/antrea-agent/agent.go

```diff
@@ -147,6 +150,7 @@ func run(o *Options) error {
 	ofClient := openflow.NewClient(o.config.OVSBridge, ovsBridgeMgmtAddr,
 		features.DefaultFeatureGate.Enabled(features.AntreaProxy),
```

Comment:
maybe we should consider passing enabled features as a list, instead of always growing the parameter list

Reply:
Actually we could directly consume the feature gate value in `openflow.NewClient` directly:
```
func NewClient(bridgeName string,
	mgmtAddr string,
	enableDenyTracking bool,
	proxyAll bool,
	connectUplinkToBridge bool) Client {
	bridge := binding.NewOFBridge(bridgeName, mgmtAddr)
	c := &client{
		bridge:                bridge,
		enableProxy:           enableProxy,
		proxyAll:              proxyAll,
		enableAntreaPolicy:    features.DefaultFeatureGate.Enabled(features.AntreaProxy),
		...
	}
}
```
I could make the change in next release.